### ISSUE TITLE
Prawn::Font::TTF#italic_angle is untested

### ISSUE THREAD

**Issue by packetmonkey** [prawnpdf/prawn#854]
During some cleaning up I came across the [Prawn::Font::TTF#italic_angle method](https://github.com/prawnpdf/prawn/blob/master/lib/prawn/font/ttf.rb#L116). As part of my normal routine I commented it out before editing the method to ensure I would see a test failure, and I did not.

The behavior of this method needs to be covered by a spec at some point, so I am leaving this open issue as a reminder.

**Comment by wpolicarpo** [comment]
Would not be the case of configuring a coverage tool to help us tracking these situations? Code Climate, perhaps?

**Comment by packetmonkey** [comment]
Good call, I actually have been meaning to check out https://coveralls.io for code coverage as well, but Code Climate could also work.

There is simplecov usage in the spec_helper, but I always forget it's there, which makes it an ideal thing to tie into CI.

If you feel up to it give it a shot! This is probably something we should do just to see if there are any more gaps in coverage.
